distrho/src/DistrhoUI.cpp

```diff
@@ -161,6 +161,9 @@ void UI::uiReshape(uint width, uint height)
     glViewport(0, 0, static_cast<GLsizei>(width), static_cast<GLsizei>(height));
     glMatrixMode(GL_MODELVIEW);
     glLoadIdentity();
+#else
+    (void)width;
```

Comment:
To follow the style of the rest of DPF code, this needs:
```
return;

// unused
(void)width;
...
```

Reply:
Is `return` mandatory? I find some other sources which use `return`, some which do not.
I can't help but feel upset when one conditional branch has `return`, and the other not.